sim/src/main/makefrag/midasexamples/Makefrag

```diff
@@ -30,15 +30,12 @@ else
 	firesim_sbt_project := {file:${firesim_base_dir}/}firesim
 endif
 
-submodules = \
-    $(addprefix $(firesim_base_dir)/,. midas midas/targetutils firesim-lib) \
-    $(addprefix $(chipyard_dir)/, \
-		$(addprefix generators/, rocket-chip) \
-		$(addprefix tools/, firrtl chisel3))
+chisel_src_dirs = \
+		$(addprefix $(firesim_base_dir)/,. midas midas/targetutils firesim-lib) \
```

Comment:
So `firesim_base_dir` actually means `<firesim_root>/sim`?

Reply:
Yeah, unfortunately. It's really the location from which you invoke sbt. sbt_base_dir.